src/config/config.go

```diff
@@ -41,38 +41,61 @@ func LoadConfig(filename string) (*Configuration, error) {
 	return &config, nil
 }
 
+// checks if each value in the config file is valid input
 func ValidateConfiguration(config *Configuration) error {
```

Comment:
nit pick ==> can be lowercase validateConfiguration because it shouldnt be called outside the package. 

Also maybe call it validateConfig to match naming convention of prior function.

Reply:
This also feels like it should only be callable by a config object. 

So maybe: 

func (c *Config) validateConfig() {
 

}

Reply:
Validate Config is called outside of config.go - in main.go after the config is loaded

Reply:
also, adding in  (c *Config) seems to be preventing it to be called? not sure why